sequencer/src/block/payload.rs

```diff
@@ -789,8 +750,13 @@ mod test {
         // make a fake proof for a nonexistent tx in the small block
         let tx = Transaction::new(Default::default(), Vec::new());
         let proof = queryable::gen_tx_proof_for_testing(
-            block.get_tx_table_len(),
-            block.get_tx_table_len_proof(&vid).unwrap().clone(),
+            0..block.raw_payload.len(),
+            TxTableEntry::from_usize(TxTable::get_tx_table_len(&block.raw_payload)),
+            vid.payload_proof(
+                &block.raw_payload,
+                0..std::cmp::min(TxTableEntry::byte_len(), block.raw_payload.len()),
+            )
+            .unwrap(), //block.get_tx_table_len_proof(&vid).unwrap().clone(),
```

Comment:
Is this comment left on purpose?

Reply:
No, thanks for spotting this.
should be fixed: [a154708](https://github.com/EspressoSystems/espresso-sequencer/pull/1212/commits/a154708834804c74e6597a50a34d8672fca2551a)